cmd/gardener-extension-provider-azure/app/app.go

```diff
@@ -263,8 +263,14 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not add ready check for webhook server to manager: %w", err)
 			}
 
-			if err := mgr.Start(ctx); err != nil {
-				return fmt.Errorf("error running manager: %w", err)
```

Comment:
@georgibaltiev can you restore the `start` function that I assume was incorrectly removed ?